launchdarkly-server-sdk/src/events/dispatcher.rs

```diff
@@ -219,30 +219,56 @@ impl EventDispatcher {
                     self.outbox.add_event(OutputEvent::FeatureRequest(inlined));
                 }
             }
-            InputEvent::Identify(identify) => {
+            InputEvent::Identify(mut identify) => {
+                if self.events_configuration.omit_anonymous_contexts {
+                    match identify.base.context.without_anonymous_contexts() {
+                        Ok(context) => identify.base.context = context,
+                        Err(e) => {
+                            error!("Failed to omit anonymous contexts: {}", e);
```

Comment:
Should we really have an error here? This seems like an expected condition.

Reply:
Fair point. I have removed it.